resources/views/transaccion_completa/mall_create.blade.php

```diff
@@ -0,0 +1,68 @@
+<!doctype html>
+<html lang="{{ str_replace('_', '-', app()->getLocale()) }}">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <title>Laravel</title>
+
+    <!-- Fonts -->
+    <link href="https://fonts.googleapis.com/css?family=Nunito:200,600" rel="stylesheet">
+
+    <style>
+        .webpay_form input {
+            font-size: 20px;
+            display: flex;
+            flex-direction:column;
+            width:50%;
+        }
+
+
+    </style>
+
+</head>
+<body>
+<h1>Transaccion Completa</h1>
+<form action="mallCreate" method="post" class="webpay_form" style="">
+    @csrf
+    <label for="buy_order">
+        Ordern de Compra
+    </label>
+    <input type="text" name="buy_order" value="ordenCompra12345678" />
+
+    <label for="session_id">
+        Id de Session
+    </label>
+    <input type="text" name="session_id" value="sesion1234564"/>
```

Comment:
Same here, you should add random values (say, a timestamp or something so the values won't be duplicated, which sometimes fails)